src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java

```diff
@@ -41,9 +41,10 @@
     private static final Logger LOG = LoggerFactory.getLogger(KairosDBStore.class);
     private static final ObjectMapper mapper = new ObjectMapper();
     private final DataServiceConfigProperties config;
+    private final Set<String> entityTagFields;
 
-    private final static Set<String> TAG_FIELDS = new HashSet<>(
-        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "kube_service_name"));
+    private final static Set<String> DEFAULT_ENTITY_TAG_FIELDS = new HashSet<>(
+        Arrays.asList("application_id", "application_version", "stack_name", "stack_version", "application","version","account_alias"));
```

Comment:
I suggest changing `account_alias` to `cluster_alias` (If I understand it correctly). See my comment here: https://github.bus.zalan.do/teapot/issues/issues/250#issuecomment-520835